docs/static/breaking-changes.asciidoc

```diff
@@ -1,310 +1,38 @@
 
 [float]
-==== URL Changes for DEB/RPM Packages
-
-The previous `packages.elastic.co` URL has been altered to `artifacts.elastic.co`.
-Ensure you update your repository files before running the upgrade process, or
-your operating system may not see the new packages.
+==== RPM/Deb package changes
 
+* For `rpm` and `deb` release artifacts, Logstash only globs config files matching `*.conf` pattern in the `conf.d` folder. 
+  Any config files which don't follow this pattern will not be loaded.
+  
 [float]
-==== Release Packages
-
-When Logstash 5.0 is installed via DEB or RPM packages, it now uses `/usr/share/logstash` to
-install binaries. Previously it used to install in `/opt/logstash` directory. This change was done to make the user experience consistent with other products in the Elastic Stack.
-
-[cols="3", options="header"]
-|===
-| |DEB |RPM
-|Logstash 2.x
-|`/opt/logstash`
-|`/opt/logstash`
-|Logstash 5.0
-|`/usr/share/logstash`
-|`/usr/share/logstash`
-|===
-
-A complete directory layout is described in <<dir-layout>>. This will likely impact any scripts that you may have written
-to support installing or manipulating Logstash, such as via Puppet.
-
-[float]
-==== Default Logging Level
-
-The default log severity level changed to `INFO` instead of `WARN` to match Elasticsearch. Existing logs
-(in core and plugins) were too noisy at the `INFO` level, so we audited our log messages and switched some of them to
-`DEBUG` level.
-
-You can use the new `logstash.yml` file to configure the `log.level` setting or continue to pass the new
-`--log.level` command line flag.
-
-[source,bash]
-----------------------------------
-bin/logstash --log.level warn
-----------------------------------
-
-[float]
-==== Plugin Manager Renamed
-
-`bin/plugin` has been renamed to `bin/logstash-plugin`. This occurred in Logstash 2.3 and it was mainly prevent `PATH` being
-polluted when other components of the Elastic Stack are installed on the same machine. Also, this provides a foundation
-for future change which will allow Elastic Stack packs to be installed via this script.
-
-Logstash 5.0 also adds a `remove` option, which is an alias for the now-deprecated `uninstall` option.
-
-As with earlier releases, the updated script allows both online and offline plugin installation. For example, to install a
-plugin named “my-plugin”, it’s as simple as running:
+==== Command Line Interface behavior
 
-[source,bash]
-----------------------------------
-bin/logstash-plugin install my-plugin
-----------------------------------
+* The `-e` and `-f` CLI options are now mutually exclusive. This also applies to the corresponding long form options `config.string` and 
+  `path.config`. This means config provided via `-e` will no longer be appended to the config provided via `-f`.
+* Configuration provided with `-e` or `config.string` will not be appended with `stdin` input and `stdout` output automatically.
```

Comment:
Actually, I have it wrong here. This should be `-f` and not `-e`.

@dedemorton but yes, if you provide a config with `-f`  or `-e`, we used to append it with `stdin` and `stdout`. We now only do it for `-e`.